scripts.py

```diff
@@ -38,6 +55,310 @@ def run(self):
         VIAFClient(self._db).run(self.force)
 
 
+
+class CheckContributorTitles(Script):
+    """ TODO """
+
+    def __init__(self, viaf=None):
+        self.viaf = viaf or VIAFClient(self._db)
+
+
+    def run(self, batch_size=1000):        
+        query = self._db.query(Contributor).filter(Contributor.viaf!=None).order_by(Contributor.id)
+
+        if self.log:
+            self.log.info(
+                "Processing %d contributors.", query.count()
+            )
+
+        contributors = True
+        offset = 0
+        output = "ContributorID|\tSortName|\tTitle"
+        print output.encode("utf8")
+        from core.model import dump_query
+        while contributors:
+            my_query = query.offset(offset).limit(batch_size)
+
+            #print "query=%s" % dump_query(my_query)
+            contributors = my_query.all()
+
+            for contributor in contributors:
+                self.process_contributor(contributor)
+            offset += batch_size
+            #self._db.commit()
```

Comment:
That code line is confusing -- the script is meant to scroll through the database without making any changes.  I added some docs.